layout/GamesPage.qml

```diff
@@ -36,9 +43,18 @@ anchors.centerIn: parent
                     startGame(index)
                 }
             }
-            onPressAndHold: {
-                if(mainView.small)
-                    PopupUtils.open(infoPopoverComp, thisItem, {"index":index} )
+            swipeEnabled: mainView.small
+            trailingActions: ListItemActions {
+                actions: [
+                    Action {
+                        iconName: "info"
+                        onTriggered: PopupUtils.open(infoOrRulesSheed, parent, {"index":index,"gameTitle":gameTitle,"mainText":gameInfo})
+                    },
+                    Action {
+                        iconName: "view-list-symbolic"
```

Comment:
I was a bit unsure choosing the right icon fore rules... I was looking for something that represents a list of smth (the first in particular) or something written (mainly the others...)
![immagine](https://user-images.githubusercontent.com/28359593/66686722-a5044400-ec80-11e9-9fb5-25ea98eac5f0.png) ![immagine](https://user-images.githubusercontent.com/28359593/66686752-c06f4f00-ec80-11e9-977b-7e568df84d2f.png) ![immagine](https://user-images.githubusercontent.com/28359593/66686766-cbc27a80-ec80-11e9-888f-6f7ae1222c98.png) ![immagine](https://user-images.githubusercontent.com/28359593/66686775-d0872e80-ec80-11e9-9330-8bc7917c7add.png) ![immagine](https://user-images.githubusercontent.com/28359593/66686789-d9780000-ec80-11e9-980e-0bbabd599c8c.png) ![immagine](https://user-images.githubusercontent.com/28359593/66686839-062c1780-ec81-11e9-899f-c36007962ece.png)

Reply:
the first looks like the best choice I think